src/SFA.DAS.EmployerAccounts/Services/PensionRegulatorService.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Web;
+using Newtonsoft.Json;
+using SFA.DAS.EmployerAccounts.Configuration;
+using SFA.DAS.EmployerAccounts.Interfaces;
+using SFA.DAS.EmployerAccounts.Models.PensionRegulator;
+using SFA.DAS.NLog.Logger;
+
+namespace SFA.DAS.EmployerAccounts.Services
+{
+    public class PensionRegulatorService : IPensionRegulatorService
+    {
+        private readonly EmployerAccountsConfiguration _configuration;
+        private readonly IHttpService _httpService;    
+        private readonly ILog _log;
+
+        public PensionRegulatorService(
+            IHttpServiceFactory httpServiceFactory,
+            EmployerAccountsConfiguration configuration,
+            ILog log)

```

Comment:
logger is injected but not used (this is the only thing I think needs resolving before going to testing)

Reply:
Removed injected logger.